runtime/src/lib.rs

```diff
@@ -1812,6 +1812,92 @@ impl pallet_multisig::Config for Runtime {
     type WeightInfo = weights::pallet_multisig::SubstrateWeight<Runtime>;
 }
 
+parameter_types! {
+    pub const ProxyDepositBase: Balance = dollars!(1);
+    pub const ProxyDepositFactor: Balance = dollars!(1);
+    pub const AnnouncementDepositBase: Balance = dollars!(1);
+    pub const AnnouncementDepositFactor: Balance = dollars!(1);
+}
+
+/// The type used to represent the kinds of proxying allowed.
+#[derive(
+    Copy,
+    Clone,
+    Eq,
+    PartialEq,
+    Ord,
+    PartialOrd,
+    Encode,
+    Decode,
+    RuntimeDebug,
+    MaxEncodedLen,
+    scale_info::TypeInfo,
+)]
+pub enum ProxyType {
+    Any,
+    NonTransfer,
+    Governance,
+    Staking,
+    WorkingGroups,
+}
+impl Default for ProxyType {
+    fn default() -> Self {
+        Self::Any
+    }
+}
+impl InstanceFilter<RuntimeCall> for ProxyType {
+    fn filter(&self, c: &RuntimeCall) -> bool {
+        match self {
```

Comment:
Could we add a `Referendum` proxy type?
It would allow calling only:
- `referendum.vote`
- `referendum.revealVote`
- `referendum.releaseVoteStake`
The `Governance` type is a bit more extensive

Reply:
Added in [2a2304b](https://github.com/Joystream/joystream/pull/5122/commits/2a2304b377044d531262561b4d9402d143ab1e18)